docs/products/m3db/index.rst

```diff
@@ -1,18 +1,27 @@
-M3DB Welcome
+Aiven for M3
 ============
 
-M3DB is an open source, **distributed time series database**. It's ideal for organisations that have a very large volume of metrics to handle. This platform was originally developed at Uber, so that gives an idea of the scale of metrics that it can handle.
+Aiven for M3 is a fully managed **distributed time series database**, deployable in the cloud of your choice which can bring unlimited scalability and high-availability to your Prometheus monitoring environment and other time series applications.
 
-M3DB plays nicely with other tools that might be in the stack already:
+Why M3?
+-------
+M3 is a great choice if your organization has a very large volume of metrics to handle, and it can be used as part of your observability solution. 
 
-* Collect metrics with Prometheus; M3DB is designed as a scalable storage backend for Prometheus.
-* Write metrics direct from your applications using the InfluxDB-compatible wire protocol.
-* Dashboards and query features available using Grafana and PromQL.
+M3 is highly compatible with other Aiven products for the following tasks:
+
+* To collect metrics with Prometheus: M3DB is designed as a scalable storage backend for Prometheus.
+* To write metrics directly from your applications using the InfluxDB-compatible wire protocol.
+* To create dashboards and query available features using Grafana and PromQL.
+
+Check out all the features on our `M3 product page <https://aiven.io/m3#full-feature-list/>`_. 
```

Comment:
The trailing slash at the end of the link is causing the linkcheck task to fail. It's only that there shouldn't be a `/` after `list` in the link there. From the screenshot you sent me, you can scroll up and see the one labelled "broken" - but these are tools that we can also set up on your laptop to check things as you go along and hopefully make things easier.

I'm not sure about linking out to a marketing page here, it would be less disruptive to list the M3 features instead. The "Platform features" aren't specific to M3 and aren't relevant on this page.

Reply:
I linked to the full features for M3 - as listed now on the site. I guess this is now the only place where we can link.

Reply:
Fixed the link (hopefully).